usersdk/windows.rst

```diff
@@ -0,0 +1,36 @@
+
+=======
+Windows
+=======
+
+For using the Python bindings of the OpenCMISS libraries you will need to install:
+
+   - `Python 2.7.11 64 bit <https://www.python.org/ftp/python/2.7.11/python-2.7.11.amd64.msi>`_
```

Comment:
No I need to check what we are actually building against, I meant to have it consistent at any rate.